setup.py

```diff
@@ -144,5 +144,6 @@ def find_package_data(where='.', package='',
         'django-appconf >= 1.0',
         'rcssmin == 1.0.6',
         'rjsmin == 1.1.0',
+        'six == 1.12.0',
```

Comment:
Why is this pinned to 1.12.0? 1.13.0 has been release recently.

Reply:
Because at the time of this PR, 1.13.0 had not been released yet. The release date was November 5th, some months later.